scripts/train_eq_local.sh

```
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -euo pipefail
+export PYTHONNOUSERSITE=1
+
+mkdir -p logs
+echo "Starting job on $(hostname) at $(date)"
+
+cd "$(git rev-parse --show-toplevel)"
+
+set -a
+. ./.env
```

Shellcheck (0.11.0)

[info] Not following: ./.env was not specified as input (see shellcheck -x).

(SC1091)